INSTALL

```diff
@@ -60,8 +60,10 @@ respective upstreams.
 Getting Qt5
 -----------
 
-We use Qt5 in order to only maintain one UI across platforms. Qt5.8 is the
-oldest version supported.
+We use Qt5 in order to only maintain one UI across platforms.
+
+Qt5.9.1 is the oldest version supported if ONLY building Subsurface (desktop version)
+Qt5.11 is the oldest version supported if also building Subsurface-mobile (ios/android)
```

Comment:
technically you can build Subsurface-mobile on Mac and Linux (I do that all the time for development), but for the purpose of the INSTALL file, I think this is fine. Thanks